Tests/SentryTests/Recording/SentryCrashCTests.swift

```diff
@@ -0,0 +1,282 @@
+@testable import Sentry
+import XCTest
+
+class SentryCrashCTests: XCTestCase {
+    func testOnCrash_notCrashedDuringCrashHandling_shouldWriteReportToDisk() throws {
+        // -- Arrange --
+        var appName = "SentryCrashCTests"
+            .cString(using: .utf8)!
+        let installDir = URL(fileURLWithPath: NSTemporaryDirectory())
+            .appendingPathComponent("SentryCrashCTests-\(UUID().uuidString)")
+        var installPath = installDir
+            .path
+            .cString(using: .utf8)!
+        let expectedReportsDir = installDir
+            .appendingPathComponent("Reports")
+
+        // Smoke test the existence of the directory
+        XCTAssertFalse(FileManager.default.fileExists(
+            atPath: expectedReportsDir.path
+        ))
```

Comment:
Gratuitous use of vertical space. I'd change all such code:
```suggestion
        var appName = "SentryCrashCTests".cString(using: .utf8)!
        let installDir = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent("SentryCrashCTests-\(UUID().uuidString)")
        var installPath = installDir.path.cString(using: .utf8)!
        let expectedReportsDir = installDir.appendingPathComponent("Reports")

        // Smoke test the existence of the directory
        XCTAssertFalse(FileManager.default.fileExists(atPath: expectedReportsDir.path))
```